RequestResponseModels/Ticket/Response/TicketResponse.cs

```diff
@@ -6,16 +6,16 @@
 
 namespace RequestResponseModels.Ticket.Response
 {
-    internal class TicketResponse
+    public class TicketResponse
     {
-        public TicketResponse(int id, string title, string body, string type, string priority, string component, int reporterId, DateTime createdAt, DateTime? updatedAt, byte? attachements)
+        public TicketResponse(int id, string title, string body, string type, string priority, string component, int reporterId, DateTime createdAt, DateTime? updatedAt, byte[]? attachements)
```

Comment:
1. response for what operation? all?
2. each time you retrieve the ticket are you downloading all the attachments?